<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAddFrd</class>
 <widget class="QWidget" name="QAddFrd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QAddFrd</string>
  </property>
  <property name="whatsThis">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QPushButton" name="m_searchcfirmButton">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>90</y>
         <width>151</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
      <widget class="QComboBox" name="m_searchage">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>140</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>18以下</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>18-22</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>22-30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30以上</string>
        </property>
       </item>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>100</y>
         <width>57</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>在线</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoRepeat">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QComboBox" name="m_searchsex">
       <property name="geometry">
        <rect>
         <x>159</x>
         <y>140</y>
         <width>71</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox{}



</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>不限</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>男</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>女</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="m_searchInfoEdit">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>93</y>
         <width>351</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
margin:1px;
border: 1.2px solid #2365ff;
border-radius:3px;
}
QLineEdit:hover{
margin:2px #2365ff;

}</string>
       </property>
       <property name="text">
        <string>1065420882</string>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>220</y>
         <width>551</width>
         <height>271</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2"/>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
